audio/out/ao.c

```diff
@@ -87,6 +88,9 @@ static const struct ao_driver * const audio_out_drivers[] = {
 #endif
 #if HAVE_SDL2_AUDIO
     &audio_out_sdl,
+#endif
+#if HAVE_PIPEWIRE
+    &audio_out_pipewire,
```

Comment:
Just a nitpick. Why is pipewire placed under wrappers?

Reply:
The goal was to make it explicitly opt-in, to give it some time to stabilize

Reply:
Ah, I see. Makes sense, especially with [this issue](https://gitlab.freedesktop.org/pipewire/pipewire/-/issues/1835) not being resolved.